clients/cli/src/main.rs

```diff
@@ -141,6 +152,21 @@ async fn main() -> Result<(), Box<dyn Error>> {
             let orchestrator = Box::new(OrchestratorClient::new(environment));
             register_node(node_id, &config_path, orchestrator).await
         }
+        Command::ProveFibSubprocess { inputs } => {
+            let inputs: (u32, u32, u32) = serde_json::from_str(&inputs)?;
+            match ProvingEngine::prove_fib_subprocess(&inputs) {
+                Ok(proof) => {
+                    let bytes = to_allocvec(&proof)?;
+                    let mut out = std::io::stdout().lock();
+                    out.write_all(&bytes)?;
+                    Ok(())
+                }
+                Err(e) => {
+                    eprintln!("{}", e);
+                    exit(3);
```

Comment:
```suggestion
                    exit(OOM_EXIT_CODE);
```
You can describe the constant in `consts.rs`.